### ISSUE TITLE
track Bountysource

### ISSUE THREAD

**Issue by whit537** [gratipay/inside.gratipay.com#316]
Bountysource started directly competing with us last spring, with the release of "[Salt](https://salt.bountysource.com/)." It's been growing steadily since. Before, Bountysource just did the bounty model. But, while we've had the wind [totally knocked out of us](https://medium.com/gratipay-blog/off-balanced-e3d19f2abece) by the Balanced shutdown, Bountysource has been gradually pivoting into the spot we once occupied alone: a "recurring crowdfunding platform to help developers earn their own salary from open-source."

Patreon swooped in and picked up [content creators](https://github.com/gratipay/gratipay.com/issues/737) before we could blink.

Assembly showed up with an innovative revenue-sharing model, but eventually gave up.

This is the third time I've felt a fire in my belly, and I've come to recognize it as the :fire: of competition. If we ignore Bountysource, they're going to eat our lunch. Simple as that. In fact, it may already be too late. I don't know how this is supposed to work, because I've never gotten mixed up in serious business competition before. But ... when in doubt, make a ticket! :-)

**Comment by mattbk** [comment]
We have some things going for us:
- We don't take a cut.
- Donations start at $0.01.

What else?

**Comment by whit537** [comment]
@mattbk What do we *not* have going for us? That's what's going to get us burned here.

**Comment by whit537** [comment]
Clarity.

**Comment by mattbk** [comment]
- [A shiny website](https://github.com/gratipay/gratipay.com/issues/3248)?
- Progress bars (see attached)?
- Share links (see attached)?
- Clarity is true.  I always get overwhelmed looking at sites like IndieGoGo that have so much going on, and that's what Salt looks like to me.  Let teams focus on the copy on their own sites, not have to keep updating another site in order to make yourselves look active to receive funding.

![screen shot 2015-08-20 at 7 41 30 pm](https://cloud.githubusercontent.com/assets/3729038/9399005/d385cf1c-4773-11e5-90a8-9dd802d29643.png)

**Comment by whit537** [comment]
I was thinking that Bountysource Salt has *more* clarity than us. Compare:

https://salt.bountysource.com/ (homepage!)

With:

https://gratipay.com/explore/teams/ (not even our homepage!)

**Comment by whit537** [comment]
We are casting the net wider than just open-source projects, which hurts us in that it dissipates our focus and makes us muddy. On the other hand, it means we get to have awesome hackerspaces as users! :-)

https://gratipay.com/catapultpgh/
https://gratipay.com/cyberpipe/
https://gratipay.com/sudo-room/ (and also https://gratipay.com/sudomesh/)
https://gratipay.com/totalism-hackbase/

**Comment by whit537** [comment]
I'll also say something similar to what I [said](https://twitter.com/whit537/status/501936043659390976) about Patreon a year ago: I'll cede to Bountysource when they:

 - go fully open-source,
 - implement https://gratipay.com/about/features/payroll,
 - and fund themselves on themselves.

**Comment by techtonik** [comment]
Replacing all the text with datavis is definitely a good direction.

And I think that there is no competition. Gratipay - it is not about bounties and competition. It is about designing **Open Source Economy of Gratitude**, and that means that we need to make a step further than just collecting the stuff for living. Without doubts, but.. let me create a ticket.

**Comment by mattbk** [comment]
It's interesting to see @whit537 stepping toward competition and away from what you posted, @techtonik.  

As Funkadelic say, "[free your mind and your ass will follow](https://www.youtube.com/watch?v=xiqmEibSY0I)."

**Comment by whit537** [comment]
Via the latest [Changelog](https://changelog.com/) newsletter:

![screen shot 2015-08-22 at 8 44 27 am](https://cloud.githubusercontent.com/assets/134455/9424009/1467ef7e-48aa-11e5-8d30-e53e17477040.png)

**Comment by mattbk** [comment]
I think we should focus on sustainable funding, not total collected.

**Comment by whit537** [comment]
/me changes the ticket title to be more honest.

**Comment by whit537** [comment]
Bringing this over from https://github.com/gratipay/inside.gratipay.com/issues/312#issuecomment-133207889:

![screen shot 2015-08-20 at 7 07 44 pm](https://cloud.githubusercontent.com/assets/134455/9397850/c6d89e0c-476e-11e5-958a-7acef2bcb9ad.png)
![screen shot 2015-08-20 at 7 07 38 pm](https://cloud.githubusercontent.com/assets/134455/9397853/ccba08d8-476e-11e5-875b-03e8f3a20b45.png)

**Comment by whit537** [comment]
[SIX STRATEGIES TO COMPETE IN BUSINESS](http://www.hyken.com/customer-service-3/six-strategies-to-compete-in-business/)

https://www.google.com/search?q=how+do+you+compete+in+business

**Comment by webmaven** [comment]
Simon Wardley is the most sophisticated thinker I know on the topic of competitive strategy.
http://blog.gardeviance.org/2015/03/wardley-map-set-of-useful-posts.html

Some recent posts not listed in that round-up:
http://blog.gardeviance.org/2015/05/on-evolution-maps-and-numpties.html
http://blog.gardeviance.org/2015/05/on-61-different-forms-of-gameplay.html (love the D&D alignment metaphor!)
http://blog.gardeviance.org/2015/05/five-key-indicators-for-success.html
http://blog.gardeviance.org/2015/07/the-100-day-corporate-get-fit-plan.html
http://blog.gardeviance.org/2015/08/on-future.html
http://blog.gardeviance.org/2015/08/on-platforms-and-ecosystems.html
http://blog.gardeviance.org/2015/08/on-common-fallacy-of-hypothesis-driven.html

**Comment by whit537** [comment]
![screen shot 2015-09-02 at 8 08 15 pm](https://cloud.githubusercontent.com/assets/134455/9647769/64e60834-51ae-11e5-8178-f22c58280565.png)

I think they might be out of invite-only? Can anyone confirm?

**Comment by whit537** [comment]
![screen shot 2015-09-11 at 2 52 20 am](https://cloud.githubusercontent.com/assets/134455/9808745/2c9fbb18-5830-11e5-8c0a-7da0164f4def.png)

**Comment by whit537** [comment]
They've rolled out search and tagging:

![screen shot 2015-09-11 at 2 54 51 am](https://cloud.githubusercontent.com/assets/134455/9808778/86367c84-5830-11e5-8b52-6d78705834dd.png)

**Comment by whit537** [comment]
![screen shot 2015-09-11 at 1 52 08 pm](https://cloud.githubusercontent.com/assets/134455/9822091/633ae394-588c-11e5-8ba5-e5b0cbacfb3e.png)
![screen shot 2015-09-11 at 1 53 04 pm](https://cloud.githubusercontent.com/assets/134455/9822101/753c9e8e-588c-11e5-8bcf-92c8e611ab9b.png)

**Comment by whit537** [comment]
I received Bountysource's announcement *immediately* after [pinging The Changelog](https://github.com/thechangelog/ping/issues/277). Seems like this could be about to heat up. :)

Product and brand.

### Product

Our product is weaker.

Bountysource has more money and, consequently, a better product with features like [updates](https://github.com/gratipay/gratipay.com/issues/2521) and [tagging](https://github.com/gratipay/gratipay.com/issues/3127).

### Brand

Our brand is stronger. They're trying to serve the open-source community, but **they're not open**. They skim 10% off the top. They don't openly discuss their competitive strategy. ;) And their brand is "Salt," which, along with the term "bounty," is vaguely mercenary:

> The word salary comes from the Latin word salarium, or "salt-money,” a Roman soldier's allowance for the purchase of salt.

![pile](https://cloud.githubusercontent.com/assets/134455/9822539/fd2fae7e-588e-11e5-8077-c3d714c558fe.jpg)

In contrast, Gratipay receives payments on the same terms as any other Team. We are open. And our brand is the heart coin, which is all about **punching an effing heart into the goddam global economy**, which we are in fact slowly but surely doing.

![heart-coin](https://cloud.githubusercontent.com/assets/134455/9822488/c649ace8-588e-11e5-8517-40a6bd2e409d.jpg)


### So what?

Our brand only comes into play if we can meet the table stakes. If we can stay close to Bountysource on product then the fact that we're truly open and they're not should help us a good deal in the marketing, but if we can't compete on features then it doesn't matter how open we are: at the end of the day, users want to use a good product!

Whoever reaches critical mass first wins. Patreon won hands-down with creatives. If we let Bountysource take the open-source market ... then we can keep straggling along with our die-hard fans and working part-time on Gratipay, since we don't have investor pressure (predictably, Bountysource [does](https://www.crunchbase.com/organization/bountysource)). But I'd really like to get this :airplane: off the ground here at some point. I'm up for some more huffing and puffing on the campfire to get it started, but at some point I *would* like to sit back and crack open a beer. :-)

**Comment by whit537** [comment]
> I'll also say something similar to what I [said](https://twitter.com/whit537/status/501936043659390976) about Patreon a year ago: I'll cede to Bountysource when they:
>
 - go fully open-source,
 - implement https://gratipay.com/about/features/payroll,
 - and fund themselves on themselves.

I wonder ... would Warren maybe actually go for this? :-)

**Comment by techtonik** [comment]
Let me iterate it a bit.. What if we concentrate more on people receiving money and sending it not for the service, not the product, just because of gratitude? We need a clear separation for this for FinCEH, and for people - clarity for one and confidence for the other. Sooner or later Bountysource will hit the same wall with FinCEH that we've had. And then it will be a question if it is a money-laundering and tax-evasion service or not. If somebody can solve a ticket for money and pay taxes, and somebody can do this for "bounty" and don't pay taxes, I can imagine some people seeing it as a tax evasion and raising questions.

So by concentrating on people I don't mean yet another social network - payment is all about money and keeping oneselves alive - I mean let's concentrate on how to make the flow legal, taxes paid and allow the transit passage of GPieces from one person to another through a series of exchanges. Gratipay can serve as an fund, where a distribution scheme is defined by rules that people are set personally. This guarantees that GP can get money into the system without question from authority. Then we need to make sure it is legal to get money for every country. Then we also need to be clear how the system could be abused, how authority sees that and what can we do about such external pressure.

**Comment by techtonik** [comment]
But the thing that bothers me much more is that I am not productive as a coder anymore. =/

**Comment by whit537** [comment]
Went out in The Changelog Weekly [today](http://email.changelog.com/t/t-C6E6890C7A27769F):

![screen shot 2015-09-12 at 8 27 29 am](https://cloud.githubusercontent.com/assets/134455/9831491/22bb5c12-5928-11e5-8e00-3c921662790e.png)

**Comment by whit537** [comment]
Salt started with one-off contributions. We got a huge demand for that in #5. Hmmm ...

**Comment by whit537** [comment]
Sorry, that's https://github.com/gratipay/gratipay.com/issues/5.

**Comment by whit537** [comment]
I was thinking that I'd like to go through Bountysource and do a feature comparison to see where we're lacking. Then it occurred to me to publish a one-page site comparing the two, because that would answer the question for us, *and* it could help us set the terms of public debate around openness: as we're able to close the feature gaps, the remaining "Open company?" feature will become more apparent.

I just bought `gratipay-or-bountysource.guide`. :-)

**Comment by whit537** [comment]
![screen shot 2015-09-12 at 10 45 14 am](https://cloud.githubusercontent.com/assets/134455/9832007/5e96812c-593b-11e5-80e3-e943c47ae6b5.png)

**Comment by whit537** [comment]
Idea is evolving ... I wrote this copy:

>### `$ diff gratipay bountysource`
>
With the launch of [Bountysource Salt](https://salt.bountysource.com) and the relaunch of [Gratipay 2.0](https://gratipay.com/), there are now two great options for crowdfunding open-source.
>
**Disclaimer:** *This site is owned by Gratipay.*
>
The purpose of this site is to document the feature differences between Gratipay and Bountysource, so that we at Gratipay can [stay competitive](https://github.com/gratipay/inside.gratipay.com/issues/316).

And now I'm thinking that we could/should generalize to something like this:

http://diff.gratipay.com/bountysource

In other words, evolve [See Also](http://inside.gratipay.com/appendices/see-also) into a site where we have pages comparing Gratipay with any and all competitors that we're tracking. Note that the primary audience for such a site is *ourselves*. We want to know who's out there and how we stack up. The secondary audience would be our users and prospective users.

**Comment by whit537** [comment]
Or do we just fold this down into IG?

http://inside.gratipay.com/diff/bountysource

**Comment by whit537** [comment]
Maybe we do both: use `gratipay-or-bountysource.guide` as a marketing piece, and offer IG/diff as a drill-down.

**Comment by whit537** [comment]
![screen shot 2015-09-12 at 11 46 14 am](https://cloud.githubusercontent.com/assets/134455/9832276/e5e0b758-5943-11e5-810d-3da50a587bf7.png)

**Comment by whit537** [comment]
http://inside.gratipay.com/appendices/diff/bountysource

**Comment by techtonik** [comment]
I don't like the red cross. Better have it just empty. Like to be filled later. But the table comparison is a way to go.

**Comment by whit537** [comment]
> I don't like the red cross. Better have it just empty.

Why? The red cross really drives the point home, imo.

![screen shot 2015-09-12 at 1 27 29 pm](https://cloud.githubusercontent.com/assets/134455/9832726/0cbad670-5952-11e5-8f72-471a284c6e59.png)

**Comment by techtonik** [comment]
Ok. We are talking about competition, not about Gratipay being funded on Bountysource, even though I like the idea. So, making the features that Bountysource in a clear message is good. But.. At the same time there maybe some stuff that we don't want to carry aboard, because it is hard to legalize. In that case red-cross looks like it is something critical.

**Comment by whit537** [comment]
> At the same time there maybe some stuff that we don't want to carry aboard, because it is hard to legalize. In that case red-cross looks like it is something critical.

Sorry, I'm not following you. :-( Can you rephrase this? The red crosses *are* things that we are critical of Bountysource about. No?

**Comment by whit537** [comment]
We're live! :-)

![screen shot 2015-09-12 at 2 20 55 pm](https://cloud.githubusercontent.com/assets/134455/9833104/7f42694a-5959-11e5-9b5b-a495101f758c.png)

**Comment by techtonik** [comment]
I wanted to say that for honest comparison we need to mark features missing from Gratipay with red crosses too. But it may be not **that** bad that me miss them. In fact, missing features can actually be a good thing, and there could be good reason and explanation to that. Like staying legal and guarding out users from potential problems with law.

**Comment by whit537** [comment]
> I wanted to say that for honest comparison we need to mark features missing from Gratipay with red crosses too.

The purpose of this website is to make the point that we're open and they're not, not to go into a detailed feature comparison. That's what the [diff](http://inside.gratipay.com/appendices/diff/bountysource) is for.

**Comment by techtonik** [comment]
I see. I thought what we will have only one page, but ok.

**Comment by whit537** [comment]
![screen shot 2015-09-12 at 3 04 35 pm](https://cloud.githubusercontent.com/assets/134455/9833304/99124c54-595f-11e5-814a-f31eb4d7d3de.png)

**Comment by techtonik** [comment]
!m @whit537

**Comment by whit537** [comment]
:-)

**Comment by whit537** [comment]
> - Progress bars (see attached)?
- Share links (see attached)?

@mattbk Noted on http://inside.gratipay.com/appendices/see-also/bountysource.

**Comment by whit537** [comment]
Pretty interesting search results. :-)

![screen shot 2015-09-12 at 3 49 02 pm](https://cloud.githubusercontent.com/assets/134455/9833465/d68e905a-5965-11e5-93fc-a023bd23652c.png)

**Comment by whit537** [comment]
Google Trends benchmark lol

![screen shot 2015-09-12 at 4 02 03 pm](https://cloud.githubusercontent.com/assets/134455/9833523/a719e8ae-5967-11e5-8034-e63beff08b6d.png)

**Comment by wkonkel** [comment]
Hi Chad and team!

I just noticed this discussion and wanted to respond to a few of the comments.

> Bountysource started directly competing with us last spring, with the release of "Salt." It's been growing steadily since. Before, Bountysource just did the bounty model.

This is not accurate. We've had support for team contributions since December 2012. At first there were Kickstarter style fundraisers that would typically span 3-6 months (e.g. [Farwest](https://www.bountysource.com/teams/cowboy/fundraiser), [JS-Git](https://www.bountysource.com/teams/js-git/fundraiser), [RVM](https://www.bountysource.com/teams/rvm/fundraiser), etc). Then we added direct one-time contributions to all teams in January 2014. And in April 2015 [by request](https://github.com/bountysource/frontend/issues/547) we added recurring contributions. Because www.bountysource.com started to have too many buttons on each page, we decided to split out contributions from bounties and have two separate but connected sites: Bounties and Salt.

Perhaps it'd be more accurate to say that Gratipay started competing directly with Bountysource in May 2015 when you announced [Teams 2.0](https://medium.com/gratipay-blog/gratipay-2-0-2453d3c53077) ("Most significantly, instead of giving to other users, now you can only give to teams").

> Sooner or later Bountysource will hit the same wall with FinCEH that we've had. And then it will be a question if it is a money-laundering and tax-evasion service or not.

We've been working closely with lawyers and tax accountants for years. As far as I'm aware, we're fully compliant with all laws and regulations, both foreign and domestic.

> go fully open-source

This is certainly on our roadmap and something we'd like to do sooner rather than later.

> implement https://gratipay.com/about/features/payroll

Teams have had the ability to split payouts amongst members for quite some time, but it is currently a manual process. We have a more automated solution on our roadmap which isn't exactly the "team takes" model you describe, but it serves a similar purpose. 

> and fund themselves on themselves

We think this would be awesome! Since we haven't released our source code yet, we're hesitant to promote our own [Salt campaign](https://salt.bountysource.com/teams/bountysource), but both are in the near future.

-Warren

**Comment by whit537** [comment]
@wkonkel Thanks for chiming in! :-)

> We've had support for team contributions since December 2012.

Then you beat us there by four months or so. From https://github.com/rubygems/rubygems.org/issues/500#issuecomment-65147249: 

> Gratipay's [teams](http://gratipay.com/about/teams) feature began life in [this commit from April 11, 2013](https://github.com/gratipay/gratipay.com/commit/83c7eaa73a594f72946be868fb90fdcb0e0bd3eb#diff-1f540dca5c867f9928789a5c48fd6d98), reached its first point of stability [a few days later](https://github.com/gratipay/gratipay.com/commit/2c509a8d23a074aeacbdf00c7dee89cc32024677), got a rewrite a couple weeks later, [on May 1](https://github.com/gratipay/gratipay.com/issues/901), and then a second rewrite to its present form two months after that, [on July 1](https://github.com/gratipay/gratipay.com/issues/1100) (with a blog post [two days later](https://medium.com/gratipay-blog/team-gittip-eba0a27825b8)).

As you point out, Gratipay Teams got an overhaul in May of this year, with our relaunch.

> And in April 2015 by request we added recurring contributions.

This is the point at which I see Bountysource and Gratipay beginning to overlap to the point of "direct competition," because recurring donations have always been essential to Gittip/Gratipay since we launched on June 1, 2012. When we relaunched in May of this year, we *removed* recurring donations to individuals, we didn't *add* recurring donations to teams: we had explicit teams for two years at that point, and of course we had some users that were implicitly teams nearly since we launched (e.g., Read the Docs).

## Where to from here?

One way or another, though, we are where we are today: Bountysource and Gratipay are now in direct competition with each other. What will come of it? I don't know! It's so exciting! :-)

Here are the endgames I see:

 1. **Bountysource wins.** You are already way ahead of us on features. If you can achieve some threshold of openness (open-source all your code, drop the 10% hard fee, etc.), then you take away our [unique value proposition](http://gratipay-or-bountysource.guide/) and sweep the market.
 2. **Gratipay wins.** If we can catch up to you on features before you can catch up to us on openness, then we may be able to come from behind and take the market back from you.
 3. **Bountysource and Gratipay merge.** We come to terms while we're still roughly equal, before either of us has swept the market.
 4. **Gratipay gives up.** We run out of energy and give up, regardless of whether Bountysource has caught up on openness.
 5. **Bountysource gives up.** You run out of money and give up, regardless of whether Gratipay has caught up on functionality.

In (1) and (2) there is a clear winner: one of us runs away with the market, rendering the other irrelevant.

In (4) and (5) we both still have respectable market share, but one of us gives up.

I think (3) is an interesting option, but I'm not sure I trust you to really be open enough. :-)

- Are you going to drop the 10% hard fee when you start funding on yourself?
- Are you going to open up the ownership of Bountysource and not just its code?
- Are you going to publish your finances?
- Are @wkonkel and @rappo going to draw from Bountysource on the same terms as anyone else working on Bountysource?
- How are you going to sell this to your investors?

Here's my current ranking of options, by decreasing likelihood:

 - (1) **Bountysource wins.** In this case, Gratipay's role is to hang on long enough to try to goad you into following through on making Bountysource truly open.
 - (3) **Bountysource and Gratipay merge.** This would mean that we can come to an agreement before you run away with the market.
 - (2) **Gratipay wins.** I think you'll achieve "enough" openness before we can achieve "enough" features, but who knows? 
 - (4) or (5) I think you've got roughly as much capital (salt) as we have energy (heart), so I'm not sure either of us will give up before a) the other has swept the market or b) we've merged. ;-)

**Comment by mattbk** [comment]
I see one more:

  '6.  There's enough room for everyone, without anyone needing to win or lose.

We're caught up in the all-or-nothing mentality that is pushed everywhere else in the business world, but it seems like Gratipay wants to be different, and might as well try to transcend that as well.  The more people donating to other people for getting things done, the better--it opens up a world where more and more people can do what they love.  Gratipay needs to be able to pay the people who work on it, which means it needs more users--but those users don't have to come from Bountysource, just like Bountysource's users don't have to come from Gratipay.

Look at the number of users on both platforms: In the grand scheme of things, it's tiny.

For those who may not remember, I'll leave this here: https://medium.com/gratipay-blog/how-is-gratipay-different-from-patreon-1f828d977935

**Comment by tshepang** [comment]
:+1: @mattbk

**Comment by whit537** [comment]
!m @mattbk @tshepang 

Perhaps! :-)

**Comment by webmaven** [comment]
Two-sided marketplaces that have strong network effects tend to be winner-take-most, with competitors relegated to small niches.

OTOH, one of the reasons for that (as the size of the market grows, the market maker gains more pricing power on the spread between willing buyer and willing seller prices) doesn't apply to Gratipay.

**Comment by mattbk** [comment]
@webmaven, interesting points.

The below are my personal thoughts.  I'm totally in favor of people getting paid for work, but this explains why I'm not all excited to win by destroying everyone else ;-)

My motivations are not everyone's but there's a point at which growth for the sake of growth becomes largesse rather than necessity.  If Gratipay becomes sustainable (and stays that way) and can pay development team members a salary commensurate with their time commitment and skills, I'd consider it successful without it needing to be the big fish.

This is similar to another business I'm a part of that simply exists because myself and others involved think it's good for us and for the community.  It sustains itself and can pay a few people part-time salaries, but it's not making us oodles of money and that's not the point.

**Comment by whit537** [comment]
> The more people donating to other people for getting things done, the better--it opens up a world where more and more people can do what they love.

---------

> I'm totally in favor of people getting paid for work[.]

Yeah, I mean, ultimately whatever happens here is good for users. I think some healthy competition is okay, because without it I know I at least can end up resting on my laurels. :-)

**Comment by whit537** [comment]
It's funny, @mattbk: I expected you to be all gung-ho about competing, since you [race](http://northernplainsathletics.com/blog/race-report-maah-daah-hey-100-0). And yet here you're the one being all [hippy-dippy](https://github.com/gratipay/inside.gratipay.com/issues/319#issuecomment-140086598). ;-)

**Comment by mattbk** [comment]
I've been racing for over half my life now, so I've mellowed a bit.  It's not so much about beating other people down as helping elevate other people to higher efforts.

**Comment by whit537** [comment]
!m @mattbk

**Comment by whit537** [comment]
How long until Patreon adds a "Software" [category](https://www.patreon.com/featured)? :disappointed: :fallen_leaf: 

[![screen shot 2015-09-29 at 8 27 46 pm](https://cloud.githubusercontent.com/assets/134455/10181628/a2f3ffc4-66e8-11e5-8a20-e4c1d11ac428.png)](https://www.patreon.com/reactdx)

**Comment by whit537** [comment]
(P.S. An interesting [use of "patrons"](https://github.com/gratipay/gratipay.com/issues/265#issuecomment-9182148) ~six months before Patreon. :-)

**Comment by whit537** [comment]
Bountysource [opened up](https://github.com/bountysource/core) about five months [ago](https://github.com/bountysource/core/commit/e164474c92d557cba79675d2f3201787a7c5caa1). I heard from @rappo in private email this past week, and we're scheduling a call in the next week or two to discuss potential implications here.

:scream_cat: :flushed: :open_mouth:

**Comment by whit537** [comment]
Call scheduled for tomorrow at [18:00 UTC](http://www.timeanddate.com/worldclock/fixedtime.html?iso=20160405T18). Gonna do a private call with a written summary after the fact.

**Comment by whit537** [comment]
> We've got a call tomorrow with @Bountysource to catch up on how our respective efforts to fund FLOSS are unfolding: https://github.com/gratipay/inside.gratipay.com/issues/316#issuecomment-204751859.

https://twitter.com/Gratipay/status/717098156278824960

**Comment by gravyboat** [comment]
Another interesting factor that isn't included here is the multiple methods which are used to 'cash out' at Bountysource. Per their FAQ they currently support: `Check, Paypal, Bitcoin, or Ripple`. @whit537 (or whoever is running the Gratipay Twitter) and I have discussed the importance of multiple ways to pay people out.

**Comment by whit537** [comment]
Okay! Here's my summary of the call, which I vetted w/ @rappo so I'm including his responses:

> Dave,
>
Thanks for the call. I appreciate the face-to-face interaction and the well-intentioned goading, as well as the invitation to join forces. Thank you. :-)
>
Here's what I plan to post:
>
------------------
>
>> We had a friendly call, which lasted almost 1.5 hours. @rappo rightly notes that Gratipay's numbers have been flat for a year, while Salt has shown steady growth. Gratipay needs to up its game if we want to stay competitive—and if/when we're ready to cede, we're all welcome to work for Bountysource. :-)
>>
Some tidbits:
>>
 - They are going the Liberapay route re: #118 (common carrier, trying to avoid politics)
 - They are agnostic about how teams split funds (there are some on-site tools, more complex scenarios happen off-site).
 - All members of all teams on Bountysource are contractors of Bountysource, Inc. (they're a "crowdsourced software company", not a payment processor).
 - They are keeping the 10% hard fee (despite having their own Salt campaign).
 - "Except for salaries, we are profitable."
 - They are now open-source, and are planning to make a push soon to build the community around Bountysource itself.
 - Their investors are on-board with this new direction.

Here's @rappo's response:

> Chad,
>
Likewise, great chatting with you and catching up. I'm serious when I say I'm rooting for Gratipay. We both have the same goals and we make each other stronger by innovating and growing :).
>
Notes look good! Two things:
>
>> They are going the Liberapay route re: #118 (common carrier, trying to avoid politics)
>
I haven't read all of #118 so I'm not sure the full extent of what this implies, but it should go without saying that all projects need to be open source and that they must not be illegal in nature. Eg the software behind the Silk Road? totally fine. A campaign to run a darknet market on Tor? not cool.
>
>> They are keeping the 10% hard fee (despite having their own Salt campaign).
>
I would clarify this as "For now, they are...". I'm open to discussion on changing the fee structure (or removing it completely if it seems feasible), but we have much bigger fish to fry right now.

**Comment by nobodxbodon** [comment]
Though I didn't do any survey myself, I feel gratipay doesn't have its due reputation in OS world yet. Gratipay is still the most open one in the market as far as I see, but I think that part is far from being highlighted and taken full advantage of, especially when marketing and propagation. For example, some breakdown of the income & costs, more technical articles about the details of running the site,  etc. More exposure and connections may go a long way.

**Comment by nobodxbodon** [comment]
BTW BountySource seems to support closed-source project. I wonder if Gratipay also does?

**Comment by whit537** [comment]
@nobodxbodon We don't right now, but I suppose we will very soon underneath https://github.com/gratipay/inside.gratipay.com/issues/432.

**Comment by whit537** [comment]
> I feel gratipay doesn't have its due reputation in OS world yet. 

Want to help us with marketing? How can we put you to work? :-)

**Comment by rappo** [comment]
@nobodxbodon,
> BTW BountySource seems to support closed-source project. I wonder if Gratipay also does?

We (Bountysource) do not allow closed-source projects on our platform. If you know of any that have snuck on please let me know and I'll boot them right off. Is Gratipay planning to support closed source projects?

**Comment by whit537** [comment]
> Is Gratipay planning to support closed source projects?

We're in the process of [relaxing](https://github.com/gratipay/inside.gratipay.com/issues/432) our review process, which as it's headed would probably allow for closed-source projects, yes.

**Comment by nobodxbodon** [comment]
>>  I feel gratipay doesn't have its due reputation in OS world yet.

>  Want to help us with marketing? How can we put you to work? :-)

@whit537 Depending on the current marketing plan, I might help with writing tech articles to some simple code if needed. Still I'm quite new to the project and I'm realizing most of my thoughts have already been brought up more or less before, so it may be better to start with small stuff first and see how it's going.

**Comment by whit537** [comment]
> it may be better to start with small stuff first and see how it's going.

👍 

Wanna [create an onboarding ticket](http://inside.gratipay.com/big-picture/welcome) for yourself to discuss further? :-)

**Comment by mattbk** [comment]
> > Is Gratipay planning to support closed source projects?
> 
> We're in the process of [relaxing](https://github.com/gratipay/inside.gratipay.com/issues/432) our review process, which as it's headed would probably allow for closed-source projects, yes.

Specifically, three levels of closed company will be possible when TWYW returns: https://github.com/gratipay/inside.gratipay.com/issues/432#issuecomment-243245667

**Comment by strypey** [comment]
@mattbk loved your quote about sporting competition so much I quoted you on GNU Social :) Great to learn more about the nuts and bolts of both Gratipay and BountySource. Are both groups away of [this useful guide](https://github.com/nayafia/lemonade-stand) to getting paid for free code dev? Probably a good idea to keep this user informed of any changes to your platforms that may impact its accuracy (pull requests!)

**Comment by techtonik** [comment]
@nayafia works with GitHub and other corporate world with the same issue, so we are aware of the work of each other. We could use more help from lawyers to build up an open system that could transparently calculate tax and make appropriate payments. This means open access to law (which is absent in my country, and this is why I am not coding anything - with reference and open tracking it is useless).

**Comment by whit537** [comment]
> Bountysource is looking to bring new core contributors to the team. This is a great opportunity for someone to take the reigns on an amazing project and help make a difference within the open-source community.
>
> Before you get worried, please be assured that **Bountysource isn’t going anywhere**. I (rappo) will continue to lead and administrate Bountysource. Warren will step aside as lead developer as soon a new core member can take his place. Warren will stay on to provide occasional technical expertise, as well as handle some of the behind-the-scenes business-y stuff with me.

https://salt.bountysource.com/teams/bountysource/updates/5-bountysource-is-searching-for-developers-to-join-our-core-team

**Comment by rappo** [comment]
@whit537, join us! :)
